src/Symfony/Bundle/FrameworkBundle/Command/WorkflowDumpCommand.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Command;
+
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Workflow\Dumper\GraphvizDumper;
+
+/**
+ * A console command for retrieving information about services.
+ *
+ * @author Grégoire Pineau <lyrixx@lyrixx.info>
+ */
+class WorkflowDumpCommand extends ContainerAwareCommand
+{
+    /**
```

Comment:
Can you access configuration from container like you did in the extension class in the Framework extension class ?

Because we already have [a lot of commands](https://github.com/symfony/symfony/issues/15689) and for now, no way to filter except using this function, I think we got a chance to avoid this command when we don't need it.

To sum up:
* if we have a workflow configurated => of course the command should be displayed: great !
* if no workflow configurated for this application => does it make sense to display this command.. probably not.

What do you think ?